atlp-rwanda/rca-barefoot-team6-bn
# Ft register user confirm 184638320

#### What does this PR do?
 Verify registered user by sending email
 ...

#### Description of  the task to be completed?
Implementing email verification
...

#### How should this be manually tested?
Checkout to ft-register-user-confirm-184638320, npm install, and Use Postman to create user with this route: ${Basic URL}/api/users
Get that user with ${Basic URL}/api/users to view his token
Paste the token in this route ${Basic URL}/api/users/verify-email/${token}
...

#### Any background context you want to provide?
Create app password from your google account to get a password to be used as APP_PASS env variable
Ensure also that you have EMAIL variable in env file
Then you will be able to send emails to the created users
...

#### What are the relevant pivotal tracker stories?
184638371
...

#### Screenshots (if appropriate)

...
